pom.xml

```diff
@@ -1617,6 +1616,24 @@
               <fail>true</fail>
             </configuration>
           </execution>
+          <execution>
+            <id>enforce-banned-transitive-dependencies-logging</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <bannedDependencies>
+                  <excludes>
+                    <exclude>log4j:log4j</exclude>
+                  </excludes>
+                  <searchTransitive>true</searchTransitive>
+                  <message>A banned transitive logging dependency was found!</message>
+                </bannedDependencies>
```

Comment:
This code snippet is already a rule so you don't need to copy-paste the entire execution block. You can just put this snippet in the existing `enforce-banned-dependencies-logging` execution. You could also use https://maven.apache.org/enforcer/enforcer-rules/banTransitiveDependencies.html rule to make it slightly less verbose.